src/controllers/comment.js

```diff
@@ -21,10 +21,13 @@ class Comment {
    * @memberof Comment
    */
   static async getAll(request, response) {
+    responder.init(response);
```

Comment:
Hi @codeshifu, could you please shed more light about what is going on here?

Reply:
Kindly expound on this part @codeshifu

Reply:
@codeshifu 👆

Reply:
@codeshifu👆🏿

Reply:
The `responder` helper function was created as a way for us to stay consistent in the way we return success/error response. I realized the response objects being sent by each dev are inconsistent and it'll become hard when working on the front-end. So I created the `responder` that uses **Express** response under the hood.

So instead of everyone having to construct responses differently. We just have to this
```js
responder.init(response); // response here is the ExpressJS response object
const { respondWith } = responder; 
// after initializing the responder we get a function for responding to incoming request

return respondWith(message, statusCode);

```

The `repondWith` knows how to construct responses based on the statusCode being passed to it (200 is default). So all we have to do is pass a `message` as an argument and forget about how to construct the actual response.

However, though, I have removed this functionality.

Reply:
Oh, makes total sense now. 👍